addons/rounded-thumbs/profile.css

```diff
@@ -0,0 +1,26 @@
+.project.thumb img {
+  border-radius: calc(var(--roundedThumbs-roundSize) * 1px);
+}
+
+.gallery.thumb span.image {
+  border-radius: calc(var(--roundedThumbs-roundSize) * 1.35px);
+}
+
+#profile-box .player,
+.media-grid .thumb,
+.gallery.thumb span.image img {
+  border-radius: calc(var(--roundedThumbs-roundSize) * 1px);
+}
+
+.media-grid .gallery.thumb.item {
+  border-radius: calc(var(--roundedThumbs-roundSize) * 1.35px);
+}
+
+#profile-box .player [data-control="edit"],
+#lfp-change-featured {
```

Comment:
```suggestion
#profile-box .player [data-control="edit"] {
```
Making the button rounded if live featured project is enabled looks weird:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/c5aa0ecc-4a8e-401c-a966-c209c1a53f43)

Reply:
Yeah, it probably does. I'd be fine with even removing this style rule entirely.